modules/understanding-upgrade-channels.adoc

```diff
@@ -7,79 +7,89 @@
 `openshift-install` binary for a given patch level of {product-title} always
 installs that patch level.
 
-{product-title} {product-version}, which includes the upgrade from the previous 4.3
-release, has three upgrade channels to choose from:
+The updates within an Upgrade Channel are designated as one of two types. These
+Update Types are defined as:
 
-* candidate-{product-version}
-* fast-{product-version}
-* stable-{product-version}
+.  General Availability (GA) Software - {product-title} GA releases are fully
+supported and production quality.
+
+.  Release Candidate (RC) Software - {product-title} RC releases are not supported
+and should only be used to test feature acceptance and assist in qualifying the next
+version of {product-title} in your infrastructure. RC releases contain all the
+features of the product and are representative of an eventual GA release.
 
-The upgrade channels contain two types of updates:
-
-.  General Availability Software (or GA) - These versions of {product-title} are
-fully supported and are considered production quality. You may upgrade to the general
-availability release from either of the fast and stable channels.
-
-.  Release Candidate Software (or RC) - These versions of {product-title} are
-representative of the eventual general availability release and are available only
-in the candidate-{product-version} channel. The release candidate will contain all
-the features of the product. You are allowed to upgrade from a release candidate to
-another release candidate and to upgrade from a previous minor version of
-{product-title} to the current release candidate. Release candidate builds are not
-supported by Red Hat and you will not be able to upgrade from a release candidate to
-the general availability release of {product-title}. Candidates should be used to
-test feature acceptance and assist in qualifying the next version of
-{product-title} in your infrastructure.
 +                                                                                    
 [NOTE]                                                                               
 ==== 
-Release candidates differ from the nightly builds found on try.openshift.com. You
+RC releases differ from the nightly builds found on try.openshift.com. You
 cannot upgrade nightly builds to nightly builds. Nightly builds are available for
 early access to features but are not upgradable or supported.
 ==== 
 
-For GA versions, the fast and stable channels present a choice between receiving
-updates as soon as they are available or allowing Red Hat to control the rollout of
-those updates.
+{product-title} {product-version} has three Upgrade Channels to choose from:
+
+* candidate-{product-version}
+* fast-{product-version}
+* stable-{product-version}
+
+RC release versions are only available in candidate-{product-version}.
+
+GA release versions are only availble in fast-{product-version} or stable-{product-version}.
```

Comment:
I think this may be confusing.  Actually, I think the whole RC vs. GA thing may be confusing.  Trying to clarify:

* We cut releases with names.
* At the moment, sometimes those names are 4.y.z-0.nightly-YYYY-MM-DD-HHMMSS (nightlies), 4.y.z-rc.n (RCs?), and 4.y.z (GAs?).
* When they're cut, nightlies are nightlies.  They are never going to end up in any channel.
* When they're cut, RCs are not in any channel.  If we like them, we add them to the candidate channel and they become release candidates.  We will never add them to fast/stable.
* When they're cut, GAs are not in any channel.  If we like them, we add them to the candidate channel and they become release candidates.  If we still like them, we add them to fast/stable and they become GA releases.  But a release image is not automatically GA because we happened to name it 4.2.0 (or whatever).

Can we drop the whole "types of updates" thing and just talk about channels?

Reply:
I do not think this is very confusing. Can this be improved further? of course. But this is a way better improvement to what we had previously and I will be happy to take this in and then gradually improve.

Reply:
Anything available in fast/stable will also be available in candidate, so I don't like "GA release versions are only availble in fast..." (also "availble" typo).  But sure, I'm fine punting and coming back to this later.